### ISSUE TITLE
 visualization problem

### ISSUE THREAD

**Issue by dlwlrma0812** [MCZhi/GameFormer-Planner#21]
Thank you for your work. I have encountered some issues with visualization. When running run_nuboard.py and importing the nuboard file from the test_log generated by the previous test, I am able to view the scores. However, an issue arises when attempting to visualize a specific scenario, resulting in the following error:
```
File "/home/nuplan/nuplan-devkit-gameformer/nuplan/planning/nuboard/base/simulation_tile.py", line 278, in _create_initial_figure
    simulation_log = SimulationLog.load_data(simulation_file)
File "/home/nuplan/nuplan-devkit-gameformer/nuplan/planning/simulation/simulation_log.py", line 100, in load_data
    data = pickle.loads(data)
ModuleNotFoundError: No module named 'Planner'
/home/miniconda3/envs/nuplan/lib/python3.9/site-packages/bokeh/server/protocol_handler.py:101: RuntimeWarning: coroutine 'WSHandler.send_message' was never awaited
  work = connection.error(message, repr(e))
```
However, if I run the simulation and directly open localhost:5006, visualization works as expected. Could you please provide insights into the possible cause of this issue? Your assistance is greatly appreciated.

**Comment by MCZhi** [comment]
Could you let me know if you used the correct conda environment to run the nuBoard? Or what command did you use to start nuBoard?

**Comment by dlwlrma0812** [comment]
The command I used to run Nuboard was python run_nuboard.py simulation_path="/home/nuplan/nuplan-devkit-gameformer/GameFormer-Planner-main/GameFormer-Planner/testing_log/closed_loop_reactive_agents/gameformer_planner/.../". However, I think I have found the reason why it wasn't working. I need to copy the config folder from the nuplan-devkit directory, as well as the nuboard folder and run_nuboard.py file, into the GameFormer-Planner directory. Doing this should allow me to visualize specific scenarios. 
Thank you.
